README.md

```
@@ -285,9 +397,88 @@ FREE ALLOCATIONS:
+028:#LINK   :02:4b:18
+029:#ASM    :02:4a:10
+030:#PRIME  :02:33:18
+032:#LIBMRG :02:2a:10
+034:#INIT   :02:31:08
+035:#ECHO   :02:4a:08
+FREE DIRECTORIES: (228)
+FREE ALLOCATIONS: (167)
+000 001 002 003 004 005 006 007 008 009 010 011 012 013 014 015 
+etc...
+```
+|Field|Description| 
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)